utils/fs_utils.go

```diff
@@ -6,12 +6,46 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"sort"
 	"strings"
 
 	"github.com/golang/glog"
 )
 
-func GetDirectorySize(path string) (int64, error) {
+// Directory stores a representaiton of a file directory.
+type Directory struct {
+	Root    string
+	Content []string
+}
+
+type DirectoryEntry struct {
+	Name string
+	Size int64
+}
+
+type EntryDiff struct {
+	Name  string
+	Size1 int64
+	Size2 int64
+}
+
+func GetSize(path string) int64 {
+	stat, err := os.Stat(path)
+	if err != nil {
+		glog.Errorf("Could not obtain size for %s: %s", path, err)
+		return -1
```

Comment:
The next thing I was working on was modifying the human-readable output, to check for any sizes that = -1 and outputting "unknown" or "error" or something under the size column instead of -1.  The reason I didn't want to error out when the size of a particular file could not be found was to allow the rest of the analysis to be performed for the non-problematic files regardless of potential size retrieval errors.